client/src/curriculum/modules-list.scss

```diff
@@ -0,0 +1,236 @@
+@use "../ui/vars" as *;
+
```

Comment:
Overall, there are a couple of selectors and styles here that would profit from some contextualization/explanation. I'm unable to review some of the styles here, because the intentions aren't obvious.